app/components/map/map_component.jsx

```diff
@@ -5,7 +5,7 @@ import LocationPin from './LocationPin.jsx'
 import { View } from "react-native";
 import HeaderComponent from "../shared/header/header_component";
 import ToolbarComponent from "../shared/toolbar/toolbar_component";
-import { mapStyles } from "./_map";
+//import { mapStyles } from "./_map";
```

Comment:
If anyone has this file please let Jenna know

Reply:
(PR review assignment:)
What is the specific issue?